libs/bootstrap/src/__tests__/app/factories/vo/entity-omit.type.ts

```diff
@@ -0,0 +1,7 @@
+type ExtractArrayMembers<T extends readonly unknown[]> =
```

Comment:
move this file to common-types library, and make a name more general, it useful not only for entities

Reply:
test passing additional field along with the DEFAULT LIST